datasette/app.py

```diff
@@ -646,57 +711,113 @@ def _metadata_recursive_update(self, orig, updated):
                 orig[key] = upd_value
         return orig
 
-    def metadata(self, key=None, database=None, table=None, fallback=True):
```

Comment:
I'm so happy to be rid of the synchronous (but undocumented) `.metadata()` method.